<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="54047" started-at="2018-09-19T11:38:05Z" finished-at="2018-09-19T11:39:00Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="54047" started-at="2018-09-19T11:38:05Z" finished-at="2018-09-19T11:39:00Z">
      <class name="testsuite.SmartTrack_Test">
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:testsuite.SmartTrack_Test@6156496]" name="beforeMethod" is-config="true" duration-ms="10" started-at="2018-09-19T17:08:05Z" finished-at="2018-09-19T17:08:05Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void testsuite.SmartTrack_Test.SmartTrackLogin() throws java.io.IOException,java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="SmartTrackLogin()[pri:0, instance:testsuite.SmartTrack_Test@6156496]" name="SmartTrackLogin" duration-ms="53597" started-at="2018-09-19T17:08:05Z" finished-at="2018-09-19T17:08:59Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Cost Center *] but found [Cost Center *]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Cost Center *] but found [Cost Center *]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at Base.requisitionPage.Supervalu(requisitionPage.java:36)
	at testsuite.SmartTrack_Test.SmartTrackLogin(SmartTrack_Test.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SmartTrackLogin -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:testsuite.SmartTrack_Test@6156496]" name="afterMethod" is-config="true" duration-ms="169" started-at="2018-09-19T17:08:59Z" finished-at="2018-09-19T17:08:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=SmartTrackLogin status=FAILURE method=SmartTrack_Test.SmartTrackLogin()[pri:0, instance:testsuite.SmartTrack_Test@6156496] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:testsuite.SmartTrack_Test@6156496]" name="endTest" is-config="true" duration-ms="233" started-at="2018-09-19T17:08:59Z" finished-at="2018-09-19T17:09:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- testsuite.SmartTrack_Test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
